docs/source/examples/supplychain/overview.rst

```diff
@@ -0,0 +1,100 @@
+*******************
+SupplyChain Example
+*******************
+
+In this example provided the infrastructure necessary track physical objects
+thru a real world supply chain on the Sawtooth Ledger. Obviously we can not put
+physical objects in to in to the Digital Ledger. We track Records on the Ledger
+which are uniquely linked to the physical object they represent.
+
+This example provides the infrastructure necessary to track physical objects in
```

Comment:
This paragraph reads as a duplicate of the above, but seems cleaner. Suggest just deleting the above paragraph and using this one.